MacPass/MPPasswordInputController.m

```diff
@@ -44,9 +48,12 @@ @interface MPPasswordInputController ()
 @property (weak) IBOutlet NSButton *enablePasswordCheckBox;
 @property (weak) IBOutlet NSButton *unlockButton;
 @property (weak) IBOutlet NSButton *cancelButton;
+@property (weak) IBOutlet NSButton *touchIdButton;
+@property (weak) IBOutlet NSButton *touchIdEnabled;
 
 @property (copy) NSString *message;
 @property (copy) NSString *cancelLabel;
+@property (copy) NSURL *databaseFileURL;
```

Comment:
This property should not be needed, since ```MPPasswordInputController``` can retrieve the current document by accessing it's ```MPDocumentWindowController```

Take a look at ```[MPPasswordInputController _didSetKeyURL:(NSNotification *)notification]``` or ```[MPPasswordInputController _selectKeyURL]```

```
MPDocument *document = (MPDocument *)self.windowController.document;
```